cedar-policy-validator/src/cedar_schema/grammar.lalrpop

```diff
@@ -82,10 +82,18 @@ match {
 
     // other tokens
     ",", ";", ":", "::", "{", "}", "[", "]",
-    "<", ">", "=", "?",
+    "<", ">", "=", "?", "@", "(", ")",
 
 }
 
+// Annotations := {'@' Ident '(' String ')'}
+Annotation: Node<()> = {
```

Comment:
I wonder if we can start sharing some code between this grammar and the policy grammar